### PR TITLE
refactor: Introduce tapir on Pekko

### PR DESCRIPTION
**Pull request by seakayone**
Migrate all public ProjectEndpoints to Tapir

### Task Description/Number

Issue Number: DEV-

### Basic Requirements

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### PR Type

What kind of change does this PR introduce?

- [ ] fix: represents bug fixes
- [x] refactor: represents production code refactoring
- [ ] feat: represents a new feature
- [ ] docs: documentation changes (no production code change)
- [ ] chore: maintenance tasks (no production code change)
- [ ] test: all about tests: adding, refactoring tests (no production code change)
- [ ] other... Please describe:

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No
- [ ] Maybe (not 100% sure => check with FE)

### Does this PR change client-test-data?

- [ ] Yes (don't forget to update the JS-LIB team about the change)
- [ ] No

### CODE REVIEW

File: webapi/src/main/scala/org/knora/webapi/routing/HandlerMapperF.scala [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  endpoint: Endpoint[Unit, INPUT, RequestRejectedException, OUTPUT, Any],
+  handler: INPUT => Task[OUTPUT]
+)
+
+final case class HandlerMapperF()(implicit val r: zio.Runtime[Any]) {
+
+  def mapEndpointAndHandler[INPUT, OUTPUT](
+    it: EndpointAndZioHandler[INPUT, OUTPUT]
+  ): Full[Unit, Unit, INPUT, RequestRejectedException, OUTPUT, Any, Future] =
+    it.endpoint.serverLogic[Future](handlerFromZio(it.handler))
```
Comment (BalduinLandolt) [inline webapi/src/main/scala/org/knora/webapi/routing/HandlerMapperF.scala:28]: nicer name for `it`?

**Review by BalduinLandolt — APPROVED** [review]
Let's wait and see what the outcome of our discussion on the topic will be. But as it stands, looks good to me

File: webapi/src/main/scala/org/knora/webapi/routing/HandlerMapperF.scala (hunk 1 above)
Comment (seakayone) [reply webapi/src/main/scala/org/knora/webapi/routing/HandlerMapperF.scala]: done => 95d48c809544908b61bbe0e1d4749b6f80f3ce3c

File: webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsResponderADM.scala [hunk 2]
```diff
@@ -389,7 +391,7 @@ final case class ProjectsResponderADMLive(
       id <- IriIdentifier.fromString(projectIri.value).toZIO.mapError(e => BadRequestException(e.getMessage))
       keywords <- projectService
                     .findProjectKeywordsBy(id)
-                    .flatMap(ZIO.fromOption(_))
+                    .some
                     .orElseFail(NotFoundException(s"Project '${projectIri.value}' not found."))
```
Comment (mpro7) [inline webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsResponderADM.scala:395]:
```suggestion
                    .someOrFail(NotFoundException(s"Project '${projectIri.value}' not found."))
```

**Review by mpro7 — APPROVED** [review]
One minor improvement which can be omitted:

**Pull request merged**